python/orca/example/data/spark_pandas.py

```diff
@@ -51,4 +51,13 @@ def process_feature(df, awake_begin=6, awake_end=23):
     trans_data_shard = data_shard.transform_shard(process_feature)
     data2 = trans_data_shard.collect()
 
+    from bigdl.orca.data.transform import *
+    scale = MinMaxScaler(inputCol=["hours", "awake"], outputCol="x_scaled")
+    data_shard = scale.fit_transform(trans_data_shard)
+    data_shard.rdd.count()
+
+    encode = LabelEncode(inputCol="value", outputCol="encoded_value")
+    encoded_data_shard = encode.fit_transform(trans_data_shard)
+    encoded_data_shard.rdd.count()
```

Comment:
(1) change to `data_shard.compute()`
(2) instead of `compute`, maybe better to show or save the results?